### ISSUE TITLE
Border for autocomplete's dropdown is not aligning correctly with textbox

### ISSUE THREAD

**Issue by jasonody** [jasonody/NamaBeer#1]
When the autocomplete directive displays some suggestions, the dropdown's border does not align squarely with the parent textbox.

This can be recreated by opening the edit window for one of the beers and entering a couple of letters (eg. "in") into the style textbox.

<img width="287" alt="autocomplete border issue" src="https://cloud.githubusercontent.com/assets/10315179/8636273/b600ab54-280d-11e5-889e-005ffb44c2fa.PNG">

**Comment by jonathanody** [comment]
I have managed to fix your display issue. 
![auto_complete_-_updated](https://cloud.githubusercontent.com/assets/10581814/8660622/ec726ea0-29a9-11e5-8180-ac7df1466f15.JPG)

I have taken the liberty of making a slight UI enhancement - I have removed the white space around the edges of the highlighting on hover as I think it looks more effective. (See below for the current hover implementation)
![image](https://cloud.githubusercontent.com/assets/10581814/8660667/3ae03ff4-29aa-11e5-9533-93612b8c4c7c.png)

I am going to have to do some additional work to make it responsive. Let me know if you are happy with the above solution and I will look into the changes I need to make so it works with the responsive bootstrap design.
